src/dnsimple-test/fixtures/v2/api/getDomainTransfer/success.http

```diff
@@ -0,0 +1,17 @@
+HTTP/1.1 200 OK
+Server: nginx
+Date: Mon, 27 Apr 2020 19:40:00 GMT
+Content-Type: application/json; charset=utf-8
+Transfer-Encoding: chunked
+Connection: keep-alive
+Status: 200 OK
+X-RateLimit-Limit: 2400
+X-RateLimit-Remaining: 2398
+X-RateLimit-Reset: 1588019949
+ETag: W/"8404bf2739e44e9f69f3a5199466776d"
+Cache-Control: max-age=0, private, must-revalidate
+X-Request-Id: 4818d867-1f72-4619-ab46-d345e6dd25eb
+X-Runtime: 0.092854
+Strict-Transport-Security: max-age=31536000
+
+{ "data": { "id": 42, "domain_id": 2, "registrant_id": 3, "state": "cancelled", "auto_renew": false, "whois_privacy": false, "premium_price": null, "status_description": "Canceled by customer", "created_at": "2020-04-27T18:08:44Z", "updated_at": "2020-04-27T18:20:01Z" } }
```

Comment:
@duduribeiro why this fixture doesn't match https://github.com/dnsimple/dnsimple-developer/blob/master/fixtures/v2/api/getDomainTransfer/success.http?

Reply:
@weppos I needed to join the lines manually, otherwise, FixtureLoader explodes.

```
         return LoadFixture().Split(new[] { "\r\n", "\r", "\n" },
                removeEmptyLines ? StringSplitOptions.RemoveEmptyEntries : StringSplitOptions.None);
```

it is based in spliting the empty lines. If you prefer, I can work to change the FixtureLoader method.

Reply:
As I mentioned in the Java client, there's only one fixture master repository and it's https://github.com/dnsimple/dnsimple-developer/tree/master/fixtures/v2/api/getDomainTransfer. Whatever content is there, the clients MUST clone the entire fixture repo and use that.

If you manipulate the responses, you are going to face issues. In fact, https://github.com/dnsimple/dnsimple-developer/blob/master/fixtures/v2/api/getDomainTransfer/success.http is not a valid JSON response nor this file as you are using spaces whereas our server doesn't.

The fixture must reflect our response. Please regenerate the two fixtures using `curl` and the `-i -s` options:

```
curl -H 'Authorization: Bearer <token>' \
       -H 'Accept: application/json' \
       -i -s \
      https://api.dnsimple.com/v2/1010/registrar/domains/example.com/transfers/1 > path/to/file.http
```

Save the server-generated files into https://github.com/dnsimple/dnsimple-developer/blob/master/fixtures/v2/api/ without modifications, then use them in the clients.